.github/workflows/build_reusable.yml

```
@@ -112,7 +112,7 @@ jobs:
       - run: rustc --version
         if: ${{ inputs.skipNativeBuild != 'yes' || inputs.needsNextest == 'yes' || inputs.needsRust == 'yes' }}
 
-      - run: npm i -g yarn "pnpm@${PNPM_VERSION}" "turbo@${TURBO_VERSION}" "@napi-rs/cli@${NAPI_CLI_VERSION}"
+      - run: corepack prepare --activate yarn@latest && npm i -g "turbo@${TURBO_VERSION}" "@napi-rs/cli@${NAPI_CLI_VERSION}"
```

```suggestion
      - run: corepack prepare --activate yarn@1.22.19 && npm i -g "turbo@${TURBO_VERSION}" "@napi-rs/cli@${NAPI_CLI_VERSION}"
```

I think latest is going to be 3.x but npm registry has 1.x as latest tag

---

yeah just tried locally and `latest` isn't supported, maybe just `yarn@1`?

---

@ForsakenHarmony corepack doesnt allow ranges, it must be a specific version like `1.22.19`

---

@styfle works if I run it locally
https://github.com/nodejs/corepack/blob/b8a4a529319eed50983f9f2c527490d07806b1bc/sources/Engine.ts#L185